recipes/nco/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+    name: nco
+    version: 4.5.5
+
+source:
+    git_url: https://github.com/nco/nco.git
+    git_tag: 4.5.5
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - gsl
+        - antlr
+        - curl
+        - zlib
+        - hdf5
+        - libnetcdf
+        - udunits2
+        - gcc
```

Comment:
No. The "standard" `gcc` does work (but what is standard? :wink:). However, when using the gcc provided by the default channel we solve issues like https://github.com/ioos/conda-recipes/issues/676.

I can change it back to the Docker image gcc if you think this will be a problem down the road.